src/forms/ChatForm/index.jsx

```diff
@@ -9,10 +9,14 @@ export const ChatForm = ({ chatId, token, onPostMessage }) => {
   const [messageText, setMessageText] = useState('')
   const [error, setError] = useState('')
 
+  const keyDownHandler = (e) => {
+    if ((e.key === 'Enter' && !e.shiftKey) || (e.key === 'Enter' && e.ctrlKey))
+      handlePostMessage(e)
```

Comment:
If think it would be good to add additional check to make sure that `messageText` is not an empty string.

```js
if (!isEmptyString(messageText)) {
  handlePostMessage(e)
}
```

Because if you try to type 2-3 new lines into textarea and then click Enter, the form will send an empty message to backend. It would be good to cover this case. WDYT?

Reply:
I agree with what you are saying, I will change it. But I didn't bother with it because  there also exists a verificarion on backend that does not let send empty message. Will change and update PR

Reply:
Okay, no problem, keep it as it is now 👍 

I'll do it later.